lib/tiny.ex

```diff
@@ -57,6 +57,24 @@ defmodule Tiny do
   rescue _ -> raise ArgumentError
   end
 
+  @doc """
+  Safely encodes a value to JSON as iodata.
+  """
+  @spec encode_to_iodata(json, Keyword.t) ::
+    { :ok, iodata } |
+    { :error, atom }
+  def encode_to_iodata(val, opts \\ []) do
+    encode(val, [iodata: true] ++ opts)
```

Comment:
Could you please drop using the form:

```elixir
def encode_to_iodata(val, opts \\ []),
  do: encode(val, [ iodata: true ] ++ opts)
```

and then same again on the `!` version?